client/src/pages/MemberProfilePage/MemberProfilePage.js

```diff
@@ -26,212 +26,214 @@ import axios from 'axios';
-                    isCurrentUser={true}
-                    onClick={console.log("Hello")}
-                  />
-                  : '' }
-
-
-
-                </div>
-                <div></div>
-              </div>
-              <div className="Connections__list-Member">
-                <Connections
-                  userViewing={false}
-                  to="/home"
-                  users={testUsers.slice(0, 6)}
-                  extraUsers="View All"
-                />
-              </div>
-            </div>
-
-            <div className="MemberProfilePage__right-contents">
-              <UpcomingTrip
-                details={["3 Days, 2 Nights", "Resort Stay", "Children Welcome", "Guided Tours"]}
-              >
-                <h1>Book a Trip</h1>
-                <TripCardUsers src={example2} location="Hawaii" date="May 2020" />
-                <TripCardUsers src={example1} location="Puerto Rico" date="April 2019" />
-              </UpcomingTrip>
-
-              <PreviousTripsCard trips={true} link="/">
-                <h1>Previous Trips</h1>
-                <TripCardUsers src={example1} location="Puerto Rico" date="April 2019" />
-                <TripCardUsers
-                  src="https://wallpaperaccess.com/full/144067.jpg"
-                  location="Hawaii"
-                  date="May 2020"
-                />
-              </PreviousTripsCard>
-            </div>
-          </div>
-
-          <div className="MemberProfilePage__group-div-Her">
-            <GroupsList heading="Her Groups" moreGroups="View All" to="/" />
-          </div>
-
-          <Footer history={this.props.history} handle_logout={this.props.handle_logout} />
-        </div>
-      );
-
-    }
-    else {
-      window.location.href="/";
-    }
-   
-  }
+	constructor(props) {
+		super(props);
+		console.log(props);
+
+		// Default CSS class to apply to the Component
+		this.state = {
+			classList: "MemberProfilePage"
+		};
+	}
+
+	// Runs after Component is loaded in the broswer
+	componentDidMount() {
+		// axios()
+
+		// console.log('memberpage props: ', this.props);
+		if (this.props.userid !== undefined) {
+			axios.get(`${BASE_URL}/profile/${this.props.userid}`,
+				{
+					headers: {
+						Authorization: `Token ${localStorage.getItem("token")}`
+					}
+				})
+				.then(res => {
+					console.log('axios ', res.data)
+
+					this.setState({
+						protectedProfileData: {
+							first_name: res.data.first_name,
+							last_name: res.data.last_name,
+							bio: res.data.bio,
+							age: res.data.age,
+							image: res.data.image,
+							city_of_residence: res.data.city_of_residence,
+							social_media: res.data.social_media,
+							user_id: res.data.user_id
+						}
+					})
+				})
+		}
+
+		// .then(res => console.log('axios ', res.data))
+
+	}
+
+
+
+	// Runs after a component has been updated
+	componentDidUpdate() { }
+
+	// Runs right before a component is removed from the DOM
+	componentWillUnmount() { }
+
```

Comment:
Can you delete these if you're not using them?

Reply:
Will do.